pypmj/core.py

```diff
@@ -494,6 +494,7 @@ def solve(self, pp_file=None, additional_keys=None, **jcm_kwargs):
         wdir = self.working_dir()
         if not os.path.exists(wdir):
             os.makedirs(wdir)
+        self.keys['wdir'] = wdir
```

Comment:
Working dir key can be useful in pp functions to know where to store additional results.